.github/workflows/build.yml

```diff
@@ -18,9 +18,10 @@ jobs:
 
     steps:
       - name: Install Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v4
```

Comment:
Can we add a dependabot config to update them regularly as well?

Reply:
Done. added the dependabot config.